medusa/providers/torrent/html/btdb.py

```diff
@@ -0,0 +1,157 @@
+# coding=utf-8
+
+"""Provider code for BTDB."""
+
+from __future__ import unicode_literals
+
+import logging
+
+from medusa import tv
+from medusa.bs4_parser import BS4Parser
+from medusa.helper.common import convert_size
+from medusa.logger.adapters.style import BraceAdapter
+from medusa.providers.torrent.torrent_provider import TorrentProvider
+
+from requests.compat import urljoin
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+class BTDBProvider(TorrentProvider):
+    """BTDB Torrent provider."""
+
+    def __init__(self):
+        """Initialize the class."""
+        super(BTDBProvider, self).__init__('BTDB')
+
+        # Credentials
+        self.public = True
+
+        # URLs
+        self.url = 'https://btdb.to'
+        self.urls = {
+            'rss': urljoin(self.url, '/q/x264/?sort=time'),
```

Comment:
Please call this `daily` instead. (I know many parts of the code still use `rss`, but we want to move away from that.)